client/tasks.py

```diff
@@ -0,0 +1,227 @@
+"""
+Celery tasks to handle api requests of write api of NodeBB.
+"""
+from logging import getLogger
+
+from celery.task import task
+from django.conf import settings
+from django.conf import settings as django_settings
+from openedx.features.openedx_nodebb_discussion.client.categories import NodeBBCategory
+from openedx.features.openedx_nodebb_discussion.client.groups import NodeBBGroup
+from openedx.features.openedx_nodebb_discussion.client.users import NodeBBUser
+from openedx.features.openedx_nodebb_discussion.client.utils import (
+    get_category_id_from_course_id, get_group_slug_from_course_id,
+    get_group_slug_from_category_id, get_group_name_from_course_id,
+    get_nodebb_uid_from_username
+)
+
+RETRY_DELAY = django_settings.NODEBB_SETTINGS['TASK_RETRY_DELAY']
+
+log = getLogger(__name__)
+
+
+def handle_response(caller, task_name, job_type, status_code, response, entity):
```

Comment:
Add details that this function presumes that the Node.js server is running behind Nginx or similar web server.

Reply:
Done